### ISSUE TITLE
Warning when running the train.py program

### ISSUE THREAD

**Issue by SuperLuckyStar666** [SeeYouStellar/NLDDPG_DQN#1]
Hi author, when I run the train.py program with pycharm software, the program gives me an error message， I'm  a beginner now, how should I solve this problem?
The error message is as follows：

NLDDPG_DQN-master\replaymemory.py:40: UserWarning: Creating a tensor from a list of numpy.ndarrays is extremely slow. Please consider converting the list to a single numpy.ndarray with numpy.array() before converting to a tensor. (Triggered internally at C:\cb\pytorch_1000000000000\work\torch\csrc\utils\tensor_new.cpp:248.)
  return torch.tensor(state, dtype=torch.float), torch.tensor(action, dtype=torch.float), torch.tensor(reward, dtype=torch.float), torch.tensor(done, dtype=torch.int), torch.tensor(next_state, dtype=torch.float)。

NLDDPG_DQN-master\agent.py:36: UserWarning: Using a target size (torch.Size([64, 64])) that is different to the input size (torch.Size([64, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  critic_loss = F.mse_loss(current_Q, target_Q)

**Comment by ZQ-C** [comment]
at the same
